src/main/java/com/dev/cinema/exceptoin/AuthenticationException.java

```diff
@@ -0,0 +1,7 @@
+package com.dev.cinema.exceptoin;
+
+public class AuthenticationException extends RuntimeException {
```

Comment:
```suggestion
public class AuthenticationException extends Exception {
```
не зміг залогінитись - це не критично)

Reply:
Done :)